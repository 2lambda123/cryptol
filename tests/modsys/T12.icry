:module T12::Main
