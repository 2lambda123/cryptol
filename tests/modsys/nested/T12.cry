module T12 where

submodule A where
  propA = B::y > 5

submodule B where
  y : Integer
  y = 42
