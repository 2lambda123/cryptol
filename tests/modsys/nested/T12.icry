:load T12.cry
y
propA
