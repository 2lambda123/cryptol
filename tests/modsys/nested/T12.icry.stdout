Loading module Cryptol
Loading module Cryptol
Loading module T12
42
True
